models/src/test/java/io/sphere/sdk/reviews/queries/ReviewQueryModelTest.java

```diff
@@ -0,0 +1,31 @@
+package io.sphere.sdk.reviews.queries;
+
+import io.sphere.sdk.queries.Predicate;
+import io.sphere.sdk.queries.StringQueryModel;
+import io.sphere.sdk.queries.StringQuerySortingModel;
+import io.sphere.sdk.reviews.Review;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by njacinto on 21/04/15.
+ */
+public class ReviewQueryModelTest {
```

Comment:
I think this class is also obsolete since the integration tests count.

You should know, that a query is not for all fields possible and you need to verify if it is possible with an integration test.